### DISCUSSION TITLE
Grid variable dumpmasks

### DISCUSSION THREAD

**Discussion by Vlasovian** [epochpic/epoch#212] [Q&A] [answered]
Hi team

Maybe I'm being thick, but if the species dumpmask outputs stuff on a species-by-species basis, what does the addition of the no_sum (i.e., don't sum over species) dumpmask do?

Thanks in advance

**Comment by keithbennett** [comment, marked as answer]
By default a derived variable is summed over all species and you get a single output array. If you specify the "species" dumpmask then you will get separate arrays for each of the species in addition to an array containing the summed total. So for two species you will get three outputs - one for each species and one summed total. If you don't want this third array to be output then you can specify the "no_sum" dumpmask.
